src/Raven.Server/Documents/Queries/QueryRunner.cs

```diff
@@ -148,7 +148,6 @@ public TermsQueryResultServerSide ExecuteGetTermsQuery(string indexName, string
 
                 try
                 {
-                    context.CloseTransaction();
```

Comment:
ExecuteGetTermsQuery is executed by using(context.OpenReadTransaction()){} so on i = 0 the tx is already open therefore it will be used on the first iteration and we cannot close it.****